src/sentry/api/endpoints/organization_details.py

```diff
@@ -317,10 +317,12 @@ def get(self, request, organization):
                                           team should be created for.
         :auth: required
         """
+        is_skinny = request.GET.get('skinny') == '1'
```

Comment:
I would use `light`, `fast` over `skinny`. :bike:

Reply:
👍 	actually changed it on another branch, do you know if we have any prior nomenclature for this?

Reply:
I know we have a `turbo`. We use `full` or `detailed` in a few places, but I'm not able to find may references to the opposite of `full`.

Reply:
Should we just invert it?

`is_detailed = request.GET.get('detailed', '1')`

Reply:
Wouldn't that require all existing API consumers to change their parameters? The default mode should be backwards compatible imo.

Reply:
It’s light only when detailed is explicitly 0